common/app/views/fragments/headerTopNav.scala.html

```diff
@@ -184,7 +185,7 @@
             data-link-name="nav3 : veggie-burger : show" aria-controls="main-menu" tabindex="-1">
```

Comment:
This is inconsistent with other uses of `x: show` which do not have a space on the left side of the colon.